src/Db.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace GRUB;
+
+/**
+ * Class Db
+ *
+ * @return db connection
+ */
+class Db {
+    /**
+     * Creates a database connection
+     *
+     * @return PDO
+     */
+    public function getDB() : \PDO
+    {
+        $db = new \PDO('mysql:host=db; dbname=ingredients', 'root', 'password');
+        $db->setAttribute(\PDO::ATTR_DEFAULT_FETCH_MODE, \PDO::FETCH_ASSOC);
+        return $db;
+    }
+
+    /**
+     * Db constructor.
+     */
+    public function __construct()
+    {
+        return $this->getDB();
+    }
```

Comment:
This constructor doesn't do anything, you cannot return a value from a constructor

Reply:
Constructor removed